packages/ui/src/coreModules/resourceManager/components/ResourceManager/MainColumn/RecordNavigationBar/index.js

```diff
@@ -13,7 +13,10 @@ const mapStateToProps = state => {
 }
 
 const propTypes = {
-  currentTableRowNumber: PropTypes.number,
+  currentTableRowNumber: PropTypes.oneOfType([
+    PropTypes.number,
+    PropTypes.object,
```

Comment:
this should not be necessary. I understand why it happened, because I wrote a line of code for you that didn't work and then as a consequence you changed this. but next time you find that you have to do this (combine two different types that normally do not mix) then you should ask before you continue :-)

see later comment with explanation